services/web/server/src/simcore_service_webserver/projects/_ports_api.py

```diff
@@ -212,7 +212,7 @@ def _get_outputs_in_workbench(workbench: dict[NodeID, Node]) -> dict[NodeID, Any
 async def _get_computation_tasks_outputs(
     app: web.Application, *, project_id: ProjectID, nodes_ids: set[NodeID]
 ) -> dict[NodeID, dict[OutputName, Any]]:
-    selection = TasksSelection(nodes_ids=nodes_ids)
+    selection = TasksSelection(nodes_ids=list(nodes_ids))
```

Comment:
I wonder which type is correct here set or list. @sanderegg

Reply:
it's a list